elliottlib/bzutil.py

```diff
@@ -1063,8 +1063,18 @@ def is_first_fix_any(bugtracker, flaw_bug, current_target_release):
         return True
 
     # filter tracker bugs by OCP product
-    tracker_bugs = [b for b in bugtracker.get_bugs(tracker_ids)
-                    if b.product == constants.BUGZILLA_PRODUCT_OCP and b.is_tracker_bug()]
+    tracker_bugs = []
+    for tracker_id in tracker_ids:
+        try:
+            b = bugtracker.get_bug(tracker_id)
```

Comment:
Getting bugs one by one could be slow. Is `get_bugs` not possible to skip certain individual errors?

Reply:
the authorized failure happened in getbugs functions from bugzilla lib, I don't think it could be modified